.github/workflows/test.yaml

```diff
@@ -104,6 +104,8 @@ jobs:
         run: make docker-build-manager CACHE_FROM=type=gha,scope=eraser-manager CACHE_TO=type=gha,scope=eraser-manager,mode=max
       - name: Build eraser
         run: make docker-build-eraser CACHE_FROM=type=gha,scope=eraser-node CACHE_TO=type=gha,scope=eraser-node,mode=max
+      - name: Build eraser for Windows
```

Comment:
let's remove this since we won't be test it yet and it will increase test duration